src/Cookie/CookieJar.php

```diff
@@ -120,7 +120,7 @@ public function clear($domain = null, $path = null, $name = null)
         } elseif (!$path) {
             $this->cookies = array_filter(
                 $this->cookies,
-                function (SetCookie $cookie) use ($path, $domain) {
+                function (SetCookie $cookie) use ($domain) {
```

Comment:
This is a BC break. It must be reverted

Reply:
Why is it a BC break?

Reply:
@Nyholm the anonymous function doesn't use the $path variable, so I don't see how this would break anything, could you elaborate please ?

Reply:
Oh sorry. I reviewed it too quickly. I thought it was the parameters to the anonymous function. 

This change is of course fine.